tests/ci/common_posix_setup.sh

```diff
@@ -221,6 +221,16 @@ function print_executable_information {
   fi
 }
 
+function sde_getenforce_check {
+  # Based on Intel SDE README, SELinux should be turned off to allow pin to work.
```

Comment:
Where do we turn this off?

Reply:
It's dependent on the host Linux kernel, so that will be left up to the host running this job. So we are going to be at the mercy of CodeBuild. It appears (at least for AL2023 SELinux is enabled in permissive mode by default), not sure what CodeBuild is using.

Reply:
Yeah it depends on the host. It's already disabled in our case, we just have an additional check here to make sure.